File: push/src/main/java/com/xiaomi/mipush/sdk/PushServiceReceiver.java
package com.xiaomi.mipush.sdk;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;

/* JADX INFO: loaded from: miuipushsdkshared_3_7_9.jar:com/xiaomi/mipush/sdk/PushServiceReceiver.class */
public class PushServiceReceiver extends BroadcastReceiver {
    @Override // android.content.BroadcastReceiver
    public void onReceive(Context context, Intent intent) {
        Intent intent2 = new Intent(context, (Class<?>) PushMessageHandler.class);
        intent2.putExtras(intent);
        intent2.setAction(intent.getAction());
        PushMessageHandler.addJob(context, intent2);
    }
}

File: legacy/src/main/java/com/xiaomi/mipush/sdk/PushServiceReceiver.kt
package com.xiaomi.mipush.sdk
import io.github.magisk317.mipush.protocol.model.*

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent

class PushServiceReceiver : BroadcastReceiver() {
    override fun onReceive(context: Context, intent: Intent) {
        val intent2 = Intent(context, PushMessageHandler::class.java)
        intent2.putExtras(intent)
        intent2.action = intent.action
        PushMessageHandler.addJob(context, intent2)
    }
}
